src/views/v2/month/event-multiday.php

```diff
@@ -15,25 +15,55 @@
 
 $event    = $this->get( 'event' );
 $event_id = $event->ID;
+$day_number = $this->get( 'day' );
 
 $classes = [ 'tribe-events-calendar-month__event-multiday' ];
+$classes_inner = [ 'tribe-events-calendar-month__event-multiday-inner' ];
 
-if ( $is_featured = tribe( 'tec.featured_events' )->is_featured( $event_id ) ) {
+// Check if it's featured.
+if ( $is_featured = isset( $event->featured ) && $event->featured ) { // @todo: later use tribe( 'tec.featured_events' )->is_featured( $event_id ) or similar
 	$classes[] = 'tribe-events-calendar-month__event-multiday--featured';
 }
+
+// If it starts today and this week, let's add the left border and set the width
+if ( $starts_today = $event->start_date == $day_number ) { // @todo:later we can check mm/dd or even year
+
+	// @todo: check if it ends this week or not, in order to split the duration
+	$classes[] = 'tribe-events-calendar-month__event-multiday-width-' . $event->duration;
+
+	if ( isset( $event->start_this_week ) && $event->start_this_week ) {
```

Comment:
`start_this_week` for the `--start` modifier is misleading, shouldn't this be where we have the `start_today` flag?

Reply:
The case in which we have a multiday event spanning across two different weeks forces us to do it that way. Let's say the week start on monday, and you have a multiday event that comes from the past week, then that event won't be starting that week: sharp edge on the left and no left border.

Reply:
No worries, I had something else in mind but this works too.